app/assets/javascripts/provider/admin/apiconfig/services/service_discovery.js

```diff
@@ -0,0 +1,107 @@
+document.addEventListener("DOMContentLoaded", (e) => {
```

Comment:
Please verify syntax for browser compatibility

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions#Browser_compatibility

Reply:
Aboutt DOMContentLoaded
If we just pu the JS at the end of the body then we do not have to wait for this event right?

Reply:
That's true, but `DOMContentLoaded` is fully supported, I think is not a problem: 
[https://caniuse.com/#feat=domcontentloaded](https://caniuse.com/#feat=domcontentloaded)